src/advanced/timestamping-tutorial.md

```diff
@@ -0,0 +1,1020 @@
+# Create Timestamping Service
+
+In this tutorial we will create a Timestamping service for an Exonum
+powered blockchain and start up a four-node network that will process requests
+to this service. The service accepts a single transaction type: submitting
+hashes of files to the blockchain. It also allows checking whether a file
+with a certain hash has been submitted to the blockchain before.
+
+The implemented service relies on the Exonum time oracle to determine the
+current time value.
+
+You can view and download the full source code of this tutorial at
+our [GitHub repository][timestamping].
+
+Prior to setting up this demo, make sure you have the necessary packages
+installed:
+
+- [git](https://git-scm.com/downloads)
+- [Rust](https://rustup.rs)
+- [Node.js & npm](https://nodejs.org/en/download/)
+
+Refer to [Cryptocurrency][Cryptocurrency] and
+[Cryptocurrency Advanced][Cryptocurrency Advanced] for our other tutorials.
```

Comment:
We need to unify the name and link to the "Cryptocurrency Advanced Demo" repository with the name of the corresponding tutorial, which is "Service with Data Proofs". I suggest changing the name of the folder in the GitHub repo and the corresponding URL address

Reply:
sure we need to accomplish it as a separate ticket. The point is that we should do it before releasing the tutorial so that all the names and links of the documents would be correct at once.